doc/material-getting-started.md

```diff
@@ -34,44 +34,91 @@ Initialization of the Material Toolkit resources is handled by the specialized `
 
 ## Installation
 
-> [!NOTE]
-> As of [Uno Platform 4.7](https://platform.uno/blog/uno-platform-4-7-new-project-template-performance-improvements-and-more/), the solution template of the Uno app has changed. There is no longer a Shared project (.shproj), it has been replaced with a regular cross-platform library containing all user code files, referred to as the **App Code Library** project. This also implies that package references can be included in a single location without the previous need to include those in all project heads.
+### Creating a new project with the Uno Material Toolkit
```

Comment:
Would be nice I think @kazo0  to point to the get started doc around the first step here:

With a note like in some other pages of the doc with:

```
> [!NOTE]
> Make sure to setup your environment first by [following our instructions](xref:Uno.GetStarted.vs2022).
```